src/Module.php

```diff
@@ -28,12 +29,13 @@ public function getConfig(): array
      */
     public function init(ModuleManager $moduleManager): void
     {
-        $event           = $moduleManager->getEvent();
-        $container       = $event->getParam('ServiceManager');
+        $event     = $moduleManager->getEvent();
+        $container = $event->getParam('ServiceManager');
```

Comment:
Is this using single quotes because the service is defined outside here?

Reply:
I think this is defined by module manager, somewhere in MVC